tests/integration/nupic/algorithms/extensive_tm_cpp_test.py

```diff
@@ -30,3 +30,7 @@
 class ExtensiveTemporalMemoryTestCPP(ExtensiveTemporalMemoryTest, unittest.TestCase):
   def getTMClass(self):
     return nupic.bindings.algorithms.TemporalMemory
+
+
+if __name__ == "__main__":
```

Comment:
We were trying to move away from this to force people to always run tests under py.test.